pom.xml

```
@@ -11,7 +11,7 @@
 
     <groupId>community.solace.spring.cloud</groupId>
     <artifactId>spring-cloud-stream-starter-request-reply</artifactId>
-    <version>5.2.0</version>
+    <version>5.2.1-SNAPSHOT</version>
```

Snapshots are not supported. 

```suggestion
    <version>5.2.1</version>
```